karma.accessibility.test.conf.js

```diff
@@ -4,11 +4,17 @@ const revisionInfo = Downloader.revisionInfo(Downloader.currentPlatform(), Chrom
 
 process.env.CHROME_BIN = revisionInfo.executablePath;
 
+/**
+ * @type {import('karma').ConfigOptions}
+ */
 var configuration = {
   singleRun: true,
-  browsers: ['ChromeHeadless'],
+  browsers: ['ChromeHeadless-Accessibility'],
   frameworks: ['jasmine'],
   browserNoActivityTimeout: 90000,
+  browserDisconnectTimeout: 120000,
+  captureTimeout: 120000,
+  processKillTimeout: 120000,
```

Comment:
These are long timeouts. If they allow you to run the tests, won't they still be very slow?

Maybe it would be worthwhile to investigate further what is preventing the full suite from running on your local setup? If it's as simple as changing the default browser to chrome, could that be the better path?

Reply:
Good point, I'll further investigate the issue.

Reply:
I tested without those timeouts and tests still run. However, even after changing my default browser and reinstalling every node package, I can't run tests without this explicit configuration of the browser at the bottom of the file.